server/form-pages/assess/makeADecision/makeADecisionTask/makeADecision.ts

```diff
@@ -65,6 +66,8 @@ export default class MakeADecision implements TasklistPage {
     const errors: TaskListErrors<this> = {}
 
     if (!this.body.decision) errors.decision = 'You must select one option'
+    if (this.body.decision && this.body.decision !== 'accept' && !this.body.rejectReason)
```

Comment:
I think this can be simplified slightly?
```suggestion
    if (this.body.decision !== 'accept' && !this.body.rejectReason)
```

Reply:
if i say this.body.decision !== 'accept' then it will give error for no decision

Reply:
I'm not sure I understand: there should be an error if there's no decision based on the line above this one. As far as I can tell, this can follow the same pattern as other pages like [type of accommodation](https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/main/server/form-pages/apply/move-on/typeOfAccommodation.ts#L58-L64), [plans in place](https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/main/server/form-pages/apply/move-on/plansInPlace.ts#L80-L92), or [placement duration](https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/main/server/form-pages/apply/move-on/placementDuration.ts#L71-L87) where the `undefined` error is handled first, then you don't need to check the value exists for the second error

Reply:
the rejectReason is only needed for rejection if you accept that is optional
so if we submit without selecting any option then you will see 2 error
1)You must select one option
2)You must enter a reason for rejection
since this.body.decision undefined and this.body.decision !== 'accept' becomes true and raise both errors
which i dont want to show thats why added this.body.decision check in the if block